### PR TITLE
Add SDXL Refiner model

### PR DESCRIPTION
**Pull request by zhanghuiyao**
1. Add SDXL-1.0-Refiner model image-to-image generative.
2. Add SDXL-1.0-Refiner LoRA fine-tune.
3. Add SDXL-1.0-PipeLine txt-to-image generative.
4. Add Multi-Aspect data process.
5. Improved sampling speed (e.q. sdxl-base sample 40 steps on Ascend910A: 125s -> 21s).
6. Adapted to MindSpore 2.1.0

### CODE REVIEW

**Review by genshimamber — APPROVED** [review]
lgtm, I will add changes to readme

File: examples/stable_diffusion_xl/README.md [hunk 1]
```diff
@@ -4,37 +4,51 @@ This folder contains [Stable Diffusion XL (SDXL)](https://arxiv.org/abs/2307.019
...
-3. Support Efficient Memory Sampling and Tuning.
-
-## Dependency
-
-- mindspore >= 2.0.1 or 2.1
-- openmpi 4.0.3 (for distributed mode)
+1. Support SDXL-1.0-Refiner model image-to-image generative.
+2. Support SDXL-1.0-Refiner LoRA fine-tune.
+3. Support SDXL-1.0-PipeLine txt-to-image generative.
+4. Support Multi-Aspect data process.
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/README.md:22]: add reference if possible

File: examples/stable_diffusion_xl/README.md [hunk 2]
```diff
@@ -4,37 +4,51 @@ This folder contains [Stable Diffusion XL (SDXL)](https://arxiv.org/abs/2307.019
...
+5. Improved sampling speed (e.q. sdxl-base sample 40 steps on Ascend910A: 125s -> 21s).
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/README.md:23]: Improved -> Faster

File: examples/stable_diffusion_xl/README.md [hunk 3]
```diff
@@ -4,37 +4,51 @@ This folder contains [Stable Diffusion XL (SDXL)](https://arxiv.org/abs/2307.019
...
 ## Getting Started
 
 See [GETTING STARTED](GETTING_STARTED.md) for details.
 
-## Notes
-
-⚠️ This function is experimental. The script fine-tunes the whole model and often times the model overfits and runs into issues like catastrophic forgetting. It's recommended to try different hyper-parameters to get the best result on your dataset.
+## Examples:
+
+Note: sample 40 steps by SDXL-1.0-Base on Ascend 910A.
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/README.md:38]: sampled

File: examples/stable_diffusion_xl/README.md [hunk 4]
```diff
@@ -4,37 +4,51 @@ This folder contains [Stable Diffusion XL (SDXL)](https://arxiv.org/abs/2307.019
...
+5. Improved sampling speed (e.q. sdxl-base sample 40 steps on Ascend910A: 125s -> 21s).
+6. Adapted to [MindSpore 2.1.0](https://www.mindspore.cn/install).
 
-To install the dependency, please run
+**Aug 30, 2023**
 
-```shell
-pip install -r requirements.txt
-```
+1. Support SDXL-1.0-Base model text-to-image generative.
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/README.md:28]: model for t2i generation?

File: examples/stable_diffusion_xl/GETTING_STARTED.md [hunk 5]
```diff
@@ -99,23 +129,30 @@ To be supplemented
 
 ## Training and Fine-Tuning
 
+⚠️ This function is experimental. The script fine-tunes the whole model and often times the model overfits and runs into issues like catastrophic forgetting. It's recommended to try different hyper-parameters to get the best result on your dataset.
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/GETTING_STARTED.md:132]: remove times

Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/GETTING_STARTED.md:132]: > times

done

File: examples/stable_diffusion_xl/README.md (hunk 4 above)
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/README.md:28]: > generation

fixed

File: examples/stable_diffusion_xl/README.md (hunk 3 above)
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/README.md:38]: > sampled

fixed

File: examples/stable_diffusion_xl/README.md [hunk 6]
```diff
@@ -4,37 +4,51 @@ This folder contains [Stable Diffusion XL (SDXL)](https://arxiv.org/abs/2307.019
...
-3. Support Efficient Memory Sampling and Tuning.
-
-## Dependency
-
-- mindspore >= 2.0.1 or 2.1
-- openmpi 4.0.3 (for distributed mode)
+1. Support SDXL-1.0-Refiner model for image-to-image generation.
+2. Support SDXL-1.0-Refiner [LoRA](https://arxiv.org/abs/2106.09685) fine-tune.
+3. Support SDXL-1.0-PipeLine for txt-to-image generation.
+4. Support Multi-Aspect data process (e.q. [sample config](./configs/training/sd_xl_base_finetune_multi_aspect.yaml)).
```
Comment (genshimamber) [inline examples/stable_diffusion_xl/README.md:22]: what is e.q.? if you are referring to "for example", its "e.g.,"

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**